index.html

```diff
@@ -0,0 +1,109 @@
+<!DOCTYPE html>
+<html lang="en">
+
+<head>
+    <meta charset="UTF-8">
+    <meta http-equiv="X-UA-Compatible" content="IE=edge">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Capstone-Project</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.2.1/css/all.min.css"
+        integrity="sha512-MV7K8+y+gLIBoVD59lQIYicR65iaqukzvf/nwasF0nqhPay5w/9lJmVM2hMDcnK1OnMGCdVK+iQrJ7lzPJQd1w=="
+        crossorigin="anonymous" referrerpolicy="no-referrer" />
+    <link rel="stylesheet" href="mobile.css">
+    <link rel="stylesheet" href="desktop.css">
+</head>
+<body>
+    <main>
+ <header>
+ <nav>
+<a class="logo" href="./index.html"><img src="./images/CC_Global_Summit_2015 logo.png" alt="img"></a>
+<ul class="desktop-ul">
+<li class="desktop-link"><a href="./about.html" class="nav-bar">About</a></li>
+<li class="desktop-link"><a href="#">Program</a></li>
+<li class="desktop-link"><a href="#">Join</a></li>
+<li class="desktop-link"><a href="#">Sponser</a></li>
+<li class="desktop-link"><a href="#">News</a></li>
+<li class="desktop-link"><a href="#" class="nav">CC Compaign</a></li>
+</ul>
+    <div class="menu-icon">
+        <div class="item-closed"><a href="#" class="closing"><i class="fa fa-times" aria-hidden="true"></i></a>
+        </div>
+        <div class="mobile-nav"><a href="./index.html" class="closing">Home</a></div>
+        <div class="mobile-nav"><a href="./about.html" class="closing">About</a></div>
+        <div class="mobile-nav"><a href="#" class="closing">Join</a></div>
+      </div>
+<div class="item-menu"><a href="#" class="openMenu"><i class="fa fa-bars"></i></a></div>
+</nav>
+</header>
+<section class="home-page">
+<p class="heading">"Hello! Sharing World"</p>
+<h1 class="big-heading">Creative Commons Global Summit 2015</h1>
+<p class="para">A joyful celebration believing in the value of openness and sharing, creating 
+a positive change with people from over 80 countries is taking place in October, in Korea.</p>
+<p class="date">2015.10.15(Thu) ~ 16(Fri)</p>
+<p class="location">@ National Museum of Korea, Art Center Nabi and more</p>
+</section>
+<section class="about">
```

Comment:
- [ ] Please make sure that your program section mobile version design is similar to the original design especially your down button as you can see on the screenshots below :point_down: 

#### Original design
![mobk](https://user-images.githubusercontent.com/6615394/210889991-90c15c91-2920-4058-b09a-95e9944af656.jpg)